README.oracle_fdw

```diff
@@ -194,39 +201,39 @@ Foreign table options
   influence on PostgreSQL query planning.
   This option is deprecated and will be removed in a future release.
 
-- max_long (optional, defaults to "32767")
+- **max_long** (optional, defaults to "32767")
 
-  The maximal length of any LONG or LONG RAW columns in the Oracle table.
+  The maximal length of any `LONG` or `LONG RAW` columns in the Oracle table.
   Possible values are integers between 1 and 1073741823 (the maximal size of a
-  bytea in PostgreSQL).  This amount of memory will be allocated at least
+  `bytea` in PostgreSQL).  This amount of memory will be allocated at least
   twice, so large values will consume a lot of memory.
-  If "max_long" is less than the length of the longest value retrieved,
-  you will receive the error message "ORA-01406: fetched column value was
-  truncated".
+  If `max_long` is less than the length of the longest value retrieved,
+  you will receive the error message `ORA-01406: fetched column value was
+  truncated`.
 
-- readonly (optional, defaults to "false")
+- **readonly** (optional, defaults to `false`)
 
-  INSERT, UPDATE and DELETE is only allowed on tables where this option is
+  `INSERT`, `UPDATE` and `DELETE` is only allowed on tables where this option is
   not set to yes/on/true.  Since these statements can only be executed from
   PostgreSQL 9.3 on, setting this option has no effect on earlier versions.
   It might still be a good idea to set it in PostgreSQL 9.2 and earlier
   on tables that you do not wish to be changed, to be prepared for an upgrade
   to PostgreSQL 9.3 or later.
 
-- sample_percent (optional, defaults to 100)
+- **sample_percent** (optional, defaults to 100)
 
-  This option only influences ANALYZE processing and can be useful to ANALYZE
-  very large tables in a reasonable time.
+  This option only influences **ANALYZE** processing and can be useful to
+  **ANALYZE** very large tables in a reasonable time.
```

Comment:
Please don't make these bold &ndash; it is a good idea to have bold reserved for options in this chapter.

If you choose to put it in backticks, you should also do that in line 191 above.